packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Providers/ModelProviders/ModelCustomizationTests.cs

```diff
@@ -59,5 +60,63 @@ public void TestCustomization_CanChangePropertyName()
             Assert.AreEqual(1, customCodeView!.Properties.Count);
             Assert.AreEqual("Prop2", customCodeView.Properties[0].Name);
         }
+
+        [Test]
+        public void TestCustomization_CanChangePropertyType()
```

Comment:
Super minor nit: I see these two functions have a lot of the same validations. Do you think there's a simple way to consolidate them ?